Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -312,6 +314,9 @@ namespace gdjs {
 
       //5) Track the movement
       this._hasReallyMoved =
+        Math.abs(object.getX() - oldX) > 0 ||
```

Comment:
Random question: is there a use case for replacing 0 by an epsilon? Like are there moment where we could have a tiny tiny displacement on the X axis? (like when against a wall?)

Reply:
Yes, it will be safer.